runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -7959,14 +7953,15 @@ void J9::Power::TreeEvaluator::genWrtbarForArrayCopy(TR::Node *node, TR::Registe
          VMCardCheckEvaluator(node, dstObjReg, cndReg, temp1Reg, temp2Reg, temp3Reg, conditions, cg);
          generateDepLabelInstruction(cg, TR::InstOpCode::label, node, generateLabelSymbol(cg), conditions);
 
-         cg->stopUsingRegister(cndReg);
-         cg->stopUsingRegister(temp1Reg);
-         cg->stopUsingRegister(temp2Reg);
-         cg->stopUsingRegister(temp3Reg);
          }
       else
          TR_ASSERT(0, "genWrtbarForArrayCopy card marking not supported for RT");
       }
+      if (conditions)
+         {
+         TR::Register *regs[1] = { dstObjReg };
```

Comment:
for 1 or 2 registers, there is a better API to use.

Reply:
Changed the code to use the simple API. The simple API is a wrapper that uses similar code to call the array version internally.